challenges/Front-End Challenge/src/App.js

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from 'react';
+import './App.css';
+import { Navbar, Footer, Form } from './components/index';
+import { loadReCaptcha } from 'react-recaptcha-v3';
+
+function App() {
+
+  useEffect(() => {
+    //This is to load the Google ReCaptcha with the sitekey
+    loadReCaptcha('6Le3WMAUAAAAAOCR9AMk7lEj-N8S_bo-nLfRKOXW');
```

Comment:
I would like to point out that this is not standard with accessing API keys. Normally you would make a secure request to a protected server to retrieve this key without giving away the value.

Reply:
> What's another way you might keep track of these keys?

Another way to keep track of these keys is to have a separate file that you import the values from. That file could be moved to the backend and simply import the value as a variable name to keep the value of it secret.